src/components/CardComponents/CompletedButton.tsx

```diff
@@ -1,24 +1,48 @@
 import React from 'react';
 import { useDispatch } from 'react-redux';
 import { setResourceIsCompleted } from '../../store/resourcesSlice';
-import RadioButtonUncheckedIcon from '@material-ui/icons/RadioButtonUnchecked';
-import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord';
+import { withStyles } from '@material-ui/core/styles';
+import { styles } from './CardButtonStyles';
+import { SvgIcon } from '@material-ui/core';
+import { ReactComponent as CheckedButton} from './checkedButton.svg';
+import { ReactComponent as UncheckedButton} from './uncheckedButton.svg';
+
 
 interface CompletedButtonProps {
     id: number;
     isCompleted: boolean;
+    classes: any;
 }
 
 function CompletedButton(props: CompletedButtonProps) {
+    const { classes } = props;
     const dispatch = useDispatch();
-    return(<div><button onClick={() => dispatch(
-        setResourceIsCompleted({
-            id: props.id,
-            isCompleted: !props.isCompleted,
-        })
-    )}>
-        {props.isCompleted ? <RadioButtonUncheckedIcon /> : <FiberManualRecordIcon />}
-    </button></div>)
+    function handleOverlay(event) {
+        event.preventDefault();
+        // event.stopPropagation();
+        dispatch(
+            setResourceIsCompleted({
+                id: props.id,
+                isCompleted: !props.isCompleted,
+            }))
+    };
+    return(<div>
+        {/* <button 
```

Comment:
could you remove the commented out portions? ty!! :D

Reply:
yes! I realized we didn't need to wrap it in a button component, since we could add a click event handler to the SvgIcon directly, which also means that we don't need to style the button to get rid of the default appearance. deleted the old button wrapper code :)